Product/Production/Services/AdminDistributionCore/src/main/java/gov/hhs/fha/nhinc/admindistribution/adapter/proxy/AdapterAdminDistributionProxyJavaImpl.java

```diff
@@ -48,7 +48,9 @@
      * @param body Emergency Message Distribution Element transaction message body received.
      * @param assertion Assertion received.
      */
-    @AdapterDelegationEvent(beforeBuilder = EDXLDistributionEventDescriptionBuilder.class, afterReturningBuilder = EDXLDistributionEventDescriptionBuilder.class, serviceType = "Admin Distribution", version = "")
+    @AdapterDelegationEvent(beforeBuilder = EDXLDistributionEventDescriptionBuilder.class,
+        afterReturningBuilder = EDXLDistributionEventDescriptionBuilder.class, serviceType = "Admin Distribution",
+        version = "LEVEL_A0")
```

Comment:
adapter-level should be `LEVEL_a0` and should be used `NhincConstants.ADAPTER_API_LEVEL.LEVEL_a0`